packages/loading-skeleton/src/LoadingSkeleton/components/LoadingInput/LoadingInput.scss

```diff
@@ -0,0 +1,32 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/border";
+
+.base {
+  background-color: rgba($color-gray-600-rgb, 0.1);
+  border-radius: calc(#{$border-solid-border-radius} * 3);
+  top: calc(#{$spacing-md} * 0.175);
+}
+
+@keyframes fadeInOut {
```

Comment:
For a future refactor, not now, I could see that we move this animation (and maybe other things?) into a central spot, because now it's duplicated for each LoadingComponent, isn't it?
Could either be a shared CSS file, or even some sort of base component. That way the animation could be tuned in one place.

Reply:
Yeah I was thinking once I've done them all to merge the duplicated styles into a central one